prisma/schema.prisma

```diff
@@ -67,8 +69,8 @@ model Addon {
   has_image   Boolean @default(false)
   is_draft    Boolean @default(true)
 
-  is_available_online  Boolean @default(false)
-  is_available_offline Boolean @default(false)
+  offlineSize Int?
+  onlineSize  Int?
```

Comment:
how will this work with the nsfw card count?

Reply:
Uhmmm it won't... forgot to add that one woops

Reply:
Added them :)